scipy/stats/tests/test_distributions.py

```diff
@@ -2274,6 +2275,63 @@ def test_trapz_vect(self):
         assert_allclose(v, res.reshape(v.shape), atol=1e-15)
 
 
+class TestArbitrary(TestCase):
```

Comment:
I think a distribution constructed via `rv_arbitrary` should be added to the test loop in `test_continuous.py`.